src/internal/clusterstate/v2.11.0/commit_totals.go

```diff
@@ -0,0 +1,39 @@
+package v2_10_0
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"github.com/jmoiron/sqlx"
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/log"
+	"github.com/pachyderm/pachyderm/v2/src/internal/migrations"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pctx"
+	"strings"
+)
+
+func normalizeCommitTotals(ctx context.Context, env migrations.Env) error {
+	ctx = pctx.Child(ctx, "normalizeCommitTotals")
+	tx := env.Tx
+	if _, err := tx.ExecContext(ctx, `ALTER TABLE pfs.commits ADD COLUMN total_fileset_id UUID REFERENCES storage.filesets(id)`); err != nil {
+		return errors.Wrap(err, "add total_fileset column to pfs.commits")
+	}
+	log.Info(ctx, "normalizing pfs.commit_totals")
+	// If there are commits in the pfs.commit_totals table that do not exist in the pfs.commits table, bail on the migration.
+	danglingCommits := []string{}
+	if err := sqlx.SelectContext(ctx, tx, &danglingCommits,
+		`SELECT t.commit_id FROM pfs.commit_totals t EXCEPT (SELECT commit_id FROM pfs.commits);`); err != nil && !errors.Is(err, sql.ErrNoRows) {
+		return errors.Wrap(err, "getting commits exclusively in pfs.commit_totals")
+	}
+	if len(danglingCommits) > 0 {
```

Comment:
I don't see the value in this code existing. If this case comes up, the user will likely have no idea. Even if they catch the log messages, there isn't anything they would do differently based on that.

Reply:
Okay, I can remove this